include/amici/vector.h

```diff
@@ -61,6 +61,10 @@ class AmiVector {
             N_VMake_Serial(static_cast<long int>(vold.vec_.size()), vec_.data());
     }
 
+    /**
+     * @brief move constructor
+     * @param other vector from which the data will be moved
```

Comment:
Does doxygen complain about those? It doesn't really add value to document the obvious...

Reply:
yep